Update index.mdx (#120)
launchdarkly/LaunchDarkly-Docs: src/content/topics/sdk/client-side/javascript/index.mdx

To learn more, read [Experimentation](/home/experimentation).

Call track with the event name to record custom conversion metrics. 

> The second argument is optional and useful for observational analytics with the use of LaunchDarkly's Data Export feature. Lastly, with Data Export, the second argument is used to give additional context without saving the data to the LaunchDarkly user context.

Reviewer: ```suggestion
The second argument is optional. It assists with observational analytics for Data Export destinations. With Data Export, the second argument gives additional context without saving the data to the LaunchDarkly user.
```
